packages/reporters/dev-server/src/HMRServer.js

```diff
@@ -130,10 +132,27 @@ export default class HMRServer {
           depsByBundle[bundle.id] = deps;
         }
 
+        let output = await asset.getCode();
+
+        if (this.options.sourceMaps) {
+          let sourcemap: ?SourceMap = await asset.getMap();
+          if (sourcemap) {
+            let sourcemapStringified = await sourcemap.stringify({
+              format: 'inline',
+              sourceRoot: '/__parcel_source_root/',
+              // TODO
+              // rootDir & fs needed for inlineSources, see PackageRunner#generateSourceMap
```

Comment:
@DeMoorJasper any thoughts what we should do here? We can't use `inlineSources` or `inline` because that's a per-target option.